applications/GeoMechanicsApplication/tests/cpp_tests/custom_processes/test_apply_initial_uniform_stress_field.cpp

```diff
@@ -131,4 +130,21 @@ KRATOS_TEST_CASE_IN_SUITE(ApplyInitialUniformStressFieldThrowsUponConstructionWh
         "'Main'. Please check the process parameters.");
 }
 
+KRATOS_TEST_CASE_IN_SUITE(CheckInfoApplyInitialUniformStressField, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    const Parameters parameters(R"({
+        "value": [1.0, 2.0, 3.0, 4.0]
+    })");
+
+    Model model;
+    auto& r_model_part = ModelSetupUtilities::CreateModelPartWithASingle2D6NDiffOrderElement(model);
+    r_model_part.GetElement(1).GetProperties()[CONSTITUTIVE_LAW] =
+        std::make_shared<GeoIncrementalLinearElasticLaw>(std::make_unique<PlaneStrain>());
```

Comment:
I have noticed that this test (and perhaps a few other ones as well) set up more things than is strictly necessary for the test to work correctly. For instance, in this case an empty model part would work equally well for the purpose of the test. By leaving the additional setup work out, the test becomes simpler, and it also becomes clearer what is strictly needed for a member function to operate correctly.
```suggestion
    auto& r_model_part = model.CreateModelPart("foo");
```

Having said that, this is a non-blocking comment. We could also improve on that later in a separate PR. I'll leave that choice up to you. :-)

Reply:
Done.